dev/generate_bom.py

```diff
@@ -49,17 +64,67 @@ class BomGenerator(Annotator):
 
   def __init__(self, options):
     self.__base_dir = options.base_dir
+    self.__docker_registry = options.docker_registry
     self.__bom_file = ''
     self.__component_versions = {}
     super(BomGenerator, self).__init__(options)
 
   @classmethod
   def init_argument_parser(cls, parser):
     """Initialize command-line arguments."""
-    parser.add_argument('--base_dir', default='.',
+    parser.add_argument('--base_dir', default='.', required=True,
                         help="Base directory containing the component's git repositories as subdirectories.")
+    parser.add_argument('--container_builder', default='gcb',
+                        help="Type of builder to use. Currently, the supported options are {'gcb', 'docker'}.")
```

Comment:
Why do we have two sets of flags, one with `container_builder={gcb, docker}`, and one with `--{google_container_builder, docker}`?